src/Pages/Main/Filter/FilterBar.js

```diff
@@ -0,0 +1,612 @@
+                                          onClick={() => addFilter(item)}
+                                        >
+                                          <ColorItemHead>
+                                            <ColorControl>
+                                              <ColorIcon
+                                                color={item.color}
+                                                data-set={item.color}
+                                              />
+                                              {item.value}
+                                            </ColorControl>
+                                          </ColorItemHead>
+                                        </ListItem>
+                                      );
+                                    })}
+                                  </ItemWrap>
+                                </ColorList>
+                              </div>
+                            </ColorContent>
+                          </section>
+                        );
+                      })}
+                    </PanelGroupControl>
+                  </DropdownPanel>
+                </DropdownContent>
+              </DropdownContainer>
+            </FilterList>
+          </FilterItem>
+        </FilterBarList>
+      </FilterBarUl>
+      <TagList>
+        {selectedList.map(item => {
+          return (
+            <Badge
+              // key={item.id + 1100}
+              color={item.color}
+              name={item.name}
+              value={value}
+              deleteFilter={deleteFilter}
+              tapId={item.tapId}
+              item={item}
+            />
+          );
+        })}
+        <TagListItem>
+          {selectedList.length !== 0 && (
+            <ListClear onClick={() => clearFilter()}>초기화</ListClear>
+          )}
+        </TagListItem>
+      </TagList>
+    </FilterBarContainer>
+  );
+}
+
+export default FilterBar;
+
+const FilterBarContainer = styled.div`
+  margin: 30px auto;
+  width: 1136px;
+  max-width: 100%;
+  min-height: 1px;
+  transition: top 0.1s;
+  z-index: 100;
+`;
+
+const FilterBarUl = styled.ul`
+  display: flex;
+`;
+
+const FilterBarList = styled.li`
+  margin: 0 3px;
+  vertical-align: bottom;
+  font-size: 0;
+  line-height: 0;
+`;
+
+const FilterItem = styled.div`
+  position: relative;
+  display: inline-block;
+`;
+
+const FilterList = styled.button`
+  padding: 7px 8px 6px;
+  background-color: #f5f5f5;
+  border-color: #f5f5f5;
+  color: #757575;
+  line-height: 19px;
+  font-size: 15px;
+  border-radius: 4px;
+`;
+
+const Image = styled.img.attrs({
+  src: '/images/Main/arrow.png',
+})`
+  width: 10px;
+  height: 10px;
+  margin-left: 5px;
+  margin-top: 5px;
+  vertical-align: -1px;
+`;
+
+const DropdownContainer = styled.div`
+  position: absolute;
+  z-index: 1000;
+  cursor: auto;
+`;
+
+const DropdownContent = styled.div`
+  display: ${props => (props.display ? 'block' : 'none')};
```

Comment:
```suggestion
  display: ${props => (props.block? 'block' : 'none')};
```
해당 부분도 콘솔창 열어서 보시면 에러가 나고 있습니다!
위와 같은 방법으로 non-boolean 속성을 변경해주세요 :0

Reply:
넵~